tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/openwire/AdvisoryOpenWireTest.java

```diff
@@ -202,5 +205,24 @@ public void testConnectionAdvisory() throws Exception {
 
    }
 
+   @Test
+   public void testXAdvisory() throws Exception {
+      final String QUEUE = "foo";
+      server.createQueue(new QueueConfiguration(QUEUE).setRoutingType(RoutingType.ANYCAST));
+
+      try (Connection advisoryConnection = factory.createConnection();
+           Connection connection = factory.createConnection()) {
+
+         advisoryConnection.start();
 
+//         final CountDownLatch numConnectionsCreatedViaAdvisoryNotificationsLatch = new CountDownLatch(19);
```

Comment:
dead code...


that's about the only thing I'm going to ask here... just remove it